docs/history.rst

```diff
@@ -1,6 +1,36 @@
 Change Log
 ==========
 
+2.2.0
+~~~~~
+* Minimum PROJ version is 6.1.0
+* `pyproj.crs` updates:
+    * Updated CRS repr (issue #264)
+    * Add Datum, CoordinateSystem, CoordinateOperation clases (issue #262)
+    * Added CRS mapping to/from CF 1.8 (pull #244)
+    * Added to_proj4_dict (issue #226)
+    * Provide option to "pretty print" WKT (issue #258)
+    * Better handle CompoundCRS (issue #265)
+    * Add consistency for `is_geographic` and `is_projected` (issue #274)
+    * Added check in CRS.to_epsg for when proj_list is null (issue #257)
+    * Disallow creation of non*CRS (eg pipeline) in CRS class (issue #267)
+    * Added support for ITRF, compound EPSG, and urn projection strings in CRS (pull #289)
+    * Remove unused _CRS.proj_version from pxd file (pull #255)
+* `pyroj.transformer` updates:
+    * Added `always_xy` option to Transformer (issue #225)
```

Comment:
Maybe add a bit of explanation here? (eg copied from the rst docs / docstring) Because I think this is actually a very important change for many users.

Reply:
Yeah, that's a good idea 👍